app/Domain/Goalcanvas/Templates/canvasDialog.blade.php

```diff
@@ -0,0 +1,182 @@
+            </select><br>
+            @else
+            <input type="hidden" name="relates" value="{{ $canvasItem['relates'] ?? array_key_first($hiddenRelatesLabels) }}">
+            @endif
+            <br>
+            <h4 class="widgettitle title-light" style="margin-bottom:0px;"><i class="fa-solid fa-ranking-star"></i> {{ __("Metrics") }}</h4>
+
+            @php
+
+            $tpl->dispatchTplEvent('beforeMeasureGoalContainer', $canvasItem); @endphp
+            <div id="measureGoalContainer">
+                <label>How will you measure this objective. What metric will you be using.</label>
+                <input type="text" name="description" value="{{ $canvasItem['description'] }}" style="width:100%"><br>
+            </div>
+
+            <div class="row">
+                <div class="col-md-3">
+                    <label>Starting Value</label>
+                    <input type="number" step="0.01" name="startValue" value="{{ $canvasItem['startValue'] }}" style="width:105px">
+                </div>
+                <div class="col-md-3">
+                    <label>Current Value</label>
+                    <input type="number" step="0.01" name="currentValue" id="currentValueField" value="{{ $canvasItem['currentValue'] }}" @if ($canvasItem['setting']=='linkAndReport' ) readonly data-tippy-content="Current value calculated from child goals" @endif style="width:105px">
+                </div>
+                <div class="col-md-3">
+                    <label>Goal Value</label>
+                    <input type="number" step="0.01" name="endValue" value="{{ $canvasItem['endValue'] }}" style="width:105px">
+                </div>
+                <div class="col-md-3">
+                    <label>Type</label>
+                    <select name="metricType">
+                        <option value="number" @if ($canvasItem['metricType']=='number' ) selected @endif>Number</option>
+                        <option value="percent" @if ($canvasItem['metricType']=='percent' ) selected @endif>% Percent</option>
+                        <option value="currency" @if ($canvasItem['metricType']=='currency' ) selected @endif>{{ __('language.currency') }}</option>
+                    </select>
+                </div>
+            </div>
+
+            <br>
+            @if ($login::userIsAtLeast($roles::$editor))
+            <input type="submit" value="{{ __("buttons.save") }}" id="primaryCanvasSubmitButton">
+            <button type="submit" class="btn btn-primary" id="saveAndClose" value="closeModal" onclick="leantime.goalCanvasController.setCloseModal();">{{ __("buttons.save_and_close") }}</button>
+            @endif
+
+            @if ($id !== '')
+            <br><br><br>
+            <input type="hidden" name="comment" value="1">
+            <h4 class="widgettitle title-light"><span class="fa fa-comments"></span>{{ __('subtitles.discussion') }}</h4>
+            @php
+            $formUrl = "/strategyPro/editCanvasItem/" . $id;
```

Comment:
Oh, seems the link is wrong. Can you replace the section with 

```

```
            @include('comments.generalComment', ['formUrl' => "/goalcanvas/editCanvasComment/" . $id .])
```